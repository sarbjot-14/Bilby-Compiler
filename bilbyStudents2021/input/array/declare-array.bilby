main{
	imm a := alloc [int](4+8);
	imm b := a[3-1];
	

}