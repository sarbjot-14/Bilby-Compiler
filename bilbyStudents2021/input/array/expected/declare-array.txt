Couldn't read file "/home/sarbjot/Desktop/cmpt379/bilbyCompiler/bilbyStudents2021/input/array/output/declare-array.asm"
